contrib/openapi/spec.yml

```diff
@@ -1,453 +1,282 @@
 swagger: '2.0'
+schemes:
+  - http
+host: api.xsnippet.org
 info:
   title: XSnippet API
-  description: REST API of XSnippet, a simple web-service for sharing code snippets
-  version: "4.0.0"
+  version: '1.0'
+  contact:
+    email: dev@xsnippet.org
+  description: |
+    The XSnippet API is organized around [REST]. Our API has predictable,
+    resource-oriented URLs, and uses HTTP response codes to indicate API
+    errors. We use built-in HTTP features, like HTTP authentication and
```

Comment:
HTTP authentication? I thought it was limited to standard methods (basic, digest etc)